pages/common/jpegoptim.md

```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
```

Comment:
In the descriptions, I'd use the capitalized form (JPEG). But that's just a preference, feel free to ignore.